frontend/src/screens/channels/IncreaseIncomingCapacity.tsx

```diff
@@ -227,10 +227,10 @@ function NewChannelInternal({
         <p className="text-muted-foreground">
           Alby Hub works with selected service providers (LSPs) which provide
           the best network connectivity and liquidity to receive payments. The
-          channel typically stays open as long as there is usage.{" "}
+          channel may stay open for a year or more as long as there is usage.{" "}
```

Comment:
as long as there is **regular** usage.

Reply:
I removed this sentence since we currently do not have alignment from all LSPs.